frontEnd/tickets-app/src/pages/SignIn/SignIn.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const SignIn = () => {
+  return (
+    <React.Fragment>
```

Comment:
you have no need in Fragment here, because here you have only one component, which returned